### PR TITLE
Don't convert text to bytes in Formatter (fix #45)

### LINKED ISSUE #45: LogstashFormatter is not working with StreamHandler
I need to output logstash formatted records to stdout for further processing via external tools. When I do it like:
```python
ch = logging.StreamHandler(stream=sys.stdout)
ch.setFormatter(LogstashFormatter())
```

I get errors that `write` method accepts `str` and not `bytes`.

Here is excerpt from `Formatter` docstring:
```
    Formatter instances are used to convert a LogRecord to text.
```

Seems that conversion to bytes is already handled in [transport](https://github.com/eht16/python-logstash-async/blob/d8b36c111d7e0f4680fca48fa25a4d150ecdb3d2/logstash_async/transport.py#L68), so I suggest to remove another one from `Formatter`.

### PR DESCRIPTION
**Pull request by sarg**

### CODE REVIEW

File: tests/formatter_test.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    def handleError(self, record):
+        self.exception = sys.exc_info()
+
+
+class LogstashFormatterTest(unittest.TestCase):
+    def test_format(self):
+        file_handler = ExceptionCatchingFileHandler(os.devnull)
+        file_handler.setFormatter(LogstashFormatter(ensure_ascii=False))
+        file_handler.emit(makeLogRecord({"msg": "тест"}))
```
Comment (eht16) [inline tests/formatter_test.py]: The non-ASCII characters break on Python2.
Making the string a unicode string on Python2 and leave as is on Python3 should be fine:
```diff
@@ -8,6 +8,8 @@
 import sys
 import unittest
 
+import six
+
 from logstash_async.formatter import LogstashFormatter
 
 
@@ -22,9 +24,10 @@
 
 class LogstashFormatterTest(unittest.TestCase):
     def test_format(self):
+        test_message = six.u("тест")
         file_handler = ExceptionCatchingFileHandler(os.devnull)
         file_handler.setFormatter(LogstashFormatter(ensure_ascii=False))
-        file_handler.emit(makeLogRecord({"msg": "тест"}))
+        file_handler.emit(makeLogRecord({"msg": test_message}))
         file_handler.close()
 
         self.assertIsNone(file_handler.exception)
```

File: logstash_async/formatter.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def _serialize(self, message):
         if sys.version_info < (3, 0):
             return json.dumps(message)
         else:
-            return bytes(json.dumps(message, ensure_ascii=self._ensure_ascii), 'utf-8')
+            return json.dumps(message, ensure_ascii=self._ensure_ascii)
```
Comment (eht16) [inline logstash_async/formatter.py]: If you like, just remove the version switch and use only your final line for the whole method.
There is no more difference to Python2 then except the `ensure_ascii` option which was simply missing for the Python2 branch.
If not, I will do it after merging.

Comment (sarg) [reply logstash_async/formatter.py]: It didn't work for me with `six.u`, so I've used `u"

Comment (eht16) [reply logstash_async/formatter.py]: Alright, thanks.

**Comment by eht16** [comment]
Thanks!

**Pull request merged**